priv/templates/phx.gen.auth/session_controller.ex

```diff
@@ -2,7 +2,37 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
   use <%= inspect context.web_module %>, :controller
 
   alias <%= inspect context.module %>
-  alias <%= inspect auth_module %>
+  alias <%= inspect auth_module %><%= if live? do %>
+
+  def create(conn, %{"_action" => "registered"} = params) do
+    do_create(conn, params, "Account created successfully!")
+  end
+
+  def create(conn, %{"_action" => "password_updated"} = params) do
+    conn
+    |> put_session(:<%= schema.singular %>_return_to, Routes.<%= schema.singular %>_settings_path(conn, :edit))
+    |> do_create(params, "Password updated successfully!")
+  end
+
+  def create(conn, params) do
+    do_create(conn, params, "Welcome back!")
+  end
+
+  defp do_create(conn, %{"<%= schema.singular %>" => <%= schema.singular %>_params}, info) do
```

Comment:
```suggestion
  def create(conn, %{"_action" => "registered"} = params) do
    create(conn, params, "Account created successfully!")
  end

  def create(conn, %{"_action" => "password_updated"} = params) do
    conn
    |> put_session(:<%= schema.singular %>_return_to, Routes.<%= schema.singular %>_settings_path(conn, :edit))
    |> create(params, "Password updated successfully!")
  end

  def create(conn, params) do
    create(conn, params, "Welcome back!")
  end

  defp create(conn, %{"<%= schema.singular %>" => <%= schema.singular %>_params}, info) do
```

Or call it `create_with_flash`?